cli/src/semgrep/commands/ci.py

```diff
@@ -139,7 +139,6 @@ def fix_head_if_github_action(metadata: GitMeta) -> None:
 @click.option(
     "--supply-chain",
     is_flag=True,
-    hidden=True,
```

Comment:
This is useless. We're not using anymore this code for semgrep ci --help anyway.
Try it! You'll see changing the value will not change anything because now we rely on osemgrep for parsing the CLI args and generating the --help.